From 9ac6875931f5f9d073ca31cd6e1fb7636afb76bc Mon Sep 17 00:00:00 2001
From: Samuli Piippo <samuli.piippo@digia.com>
Date: Thu, 23 May 2013 14:37:10 +0300
Subject: [PATCH] beagle: change default kernel args

- default resolution to 800x600
- increase VRAM for fb0
- disable console blank
- disable console cursor blinking
- use extrabootargs
---
 include/configs/omap3_beagle.h |   20 ++++++++++++++++----
 1 file changed, 16 insertions(+), 4 deletions(-)

diff --git a/include/configs/omap3_beagle.h b/include/configs/omap3_beagle.h
index 1cc46dc..945e15e 100644
--- a/include/configs/omap3_beagle.h
+++ b/include/configs/omap3_beagle.h
@@ -224,7 +224,7 @@
 	"optargs=\0" \
 	"camera=none\0" \
 	"vram=12M\0" \
-	"dvimode=640x480MR-16@60\0" \
+	"dvimode=800x600MR-16@60\0" \
 	"defaultdisplay=dvi\0" \
 	"mmcdev=0\0" \
 	"mmcroot=/dev/mmcblk0p2 ro\0" \
@@ -240,20 +240,28 @@
 		"buddy2=${buddy2} "\
 		"camera=${camera} "\
 		"vram=${vram} " \
+		"omapfb.vram=0:${vram} " \
 		"omapfb.mode=dvi:${dvimode} " \
 		"omapdss.def_disp=${defaultdisplay} " \
 		"root=${mmcroot} " \
-		"rootfstype=${mmcrootfstype}\0" \
+		"rootfstype=${mmcrootfstype} " \
+		"consoleblank=0 " \
+		"vt.global_cursor_default=0 " \
+		"${extrabootargs}\0" \
 	"nandargs=setenv bootargs console=${console} " \
 		"${optargs} " \
 		"mpurate=${mpurate} " \
 		"buddy=${buddy} "\
 		"camera=${camera} "\
 		"vram=${vram} " \
+		"omapfb.vram=0:${vram} " \
 		"omapfb.mode=dvi:${dvimode} " \
 		"omapdss.def_disp=${defaultdisplay} " \
 		"root=${nandroot} " \
-		"rootfstype=${nandrootfstype}\0" \
+		"rootfstype=${nandrootfstype} " \
+		"consoleblank=0 " \
+		"vt.global_cursor_default=0 " \
+		"${extrabootargs}\0" \
 	"bootenv=uEnv.txt\0" \
 	"loadbootenv=fatload mmc ${mmcdev} ${loadaddr} ${bootenv}\0" \
 	"importbootenv=echo Importing environment from mmc ...; " \
@@ -263,10 +271,14 @@
 		"mpurate=${mpurate} " \
 		"buddy=${buddy} "\
 		"vram=${vram} " \
+		"omapfb.vram=0:${vram} " \
 		"omapfb.mode=dvi:${dvimode} " \
 		"omapdss.def_disp=${defaultdisplay} " \
 		"root=${ramroot} " \
-		"rootfstype=${ramrootfstype}\0" \
+		"rootfstype=${ramrootfstype} " \
+		"consoleblank=0 " \
+		"vt.global_cursor_default=0 " \
+		"${extrabootargs}\0" \
 	"loadramdisk=fatload mmc ${mmcdev} ${rdaddr} ramdisk.gz\0" \
 	"loaduimagefat=fatload mmc ${mmcdev} ${loadaddr} uImage\0" \
 	"loaduimage=ext2load mmc ${mmcdev}:2 ${loadaddr} /boot/uImage\0" \
-- 
1.7.10.4

